cmd/goreleaser/internal/configure.go

```diff
@@ -43,9 +43,9 @@ func Generate(imagePrefixes []string, dists []string) config.Project {
 			NameTemplate: "{{ .ProjectName }}_checksums.txt",
 		},
 
-		Builds:          Builds(dists),
-		Archives:        Archives(dists),
-		NFPMs:           Packages(dists),
+		Builds:   Builds(dists),
+		Archives: Archives(dists),
+		//NFPMs:           Packages(dists),
```

Comment:
do we need this?

Reply:
If you mean the commented out part only, then no, it can be removed, only Docker images are published, not native packages. I will remove it in the next PR (introducing trivy), is that alright?